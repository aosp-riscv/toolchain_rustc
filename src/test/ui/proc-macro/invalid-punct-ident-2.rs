// aux-build:invalid-punct-ident.rs

// FIXME https://github.com/rust-lang/rust/issues/59998
// normalize-stderr-test "thread.*panicked.*proc_macro_server.rs.*\n" -> ""
// normalize-stderr-test "note:.*RUST_BACKTRACE=1.*\n" -> ""
<<<<<<< HEAD   (086005 Importing rustc-1.38.0)
=======
// normalize-stderr-test "\nerror: internal compiler error.*\n\n" -> ""
// normalize-stderr-test "note:.*unexpectedly panicked.*\n\n" -> ""
// normalize-stderr-test "note: we would appreciate a bug report.*\n\n" -> ""
// normalize-stderr-test "note: compiler flags.*\n\n" -> ""
// normalize-stderr-test "note: rustc.*running on.*\n\n" -> ""
>>>>>>> BRANCH (8cd2c9 Importing rustc-1.39.0)

#[macro_use]
extern crate invalid_punct_ident;

invalid_ident!(); //~ ERROR proc macro panicked
