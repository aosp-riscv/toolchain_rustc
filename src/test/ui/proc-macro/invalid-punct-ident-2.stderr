error: proc macro panicked
<<<<<<< HEAD   (086005 Importing rustc-1.38.0)
  --> $DIR/invalid-punct-ident-2.rs:10:1
=======
  --> $DIR/invalid-punct-ident-2.rs:15:1
>>>>>>> BRANCH (8cd2c9 Importing rustc-1.39.0)
   |
LL | invalid_ident!();
   | ^^^^^^^^^^^^^^^^^
   |
   = help: message: `"*"` is not a valid identifier

error: aborting due to previous error

